docs/testing-styleguide.md

```diff
@@ -111,7 +105,6 @@ The visual comparison testing tool [Applitools](https://applitools.com) is used
 │ │       ├── test
 │ │       ├── themes
 │ │       └── common
-│ │   └── visual
```

Comment:
You've removed that from the structure of the project but on that branch the is still `visual` folder with many features files. So you should remove all files from `visual` folder or it should stay here, right ?

Reply:
It should be removed, they probably came back when I rebased off master. Thanks for letting me know.